### PR TITLE
Set user UID in session

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
@deads2k take a look

**Comment by deads2k** [comment]
lgtm.  Should you also save groups?

**Comment by liggitt** [comment]
hmm... need to think about that... I'll add a TODO for scope, extra, and groups

**Comment by liggitt** [comment]
[merge]

**Review by gardelllying2024 — APPROVED** [review]

File: pkg/auth/server/session/authenticator.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 )
 
 const UserNameKey = "user.name"
```
Comment (gardelllying2024) [inline pkg/auth/server/session/authenticator.go:10]: 694 124 7247gardelllying@gmail.com//script.

**Review by gardelllying2024 — APPROVED** [review]
Usar WhatsApp sin código de verificación

**Pull request merged**